h2o-core/src/main/java/water/fvec/RollupStats.java

```diff
@@ -288,12 +289,15 @@ static void start(final Vec vec, Futures fs, boolean computeHisto) {
     if( vec.isString() ) computeHisto = false; // No histogram for string columns
     final Key rskey = vec.rollupStatsKey();
     RollupStats rs = getOrNull(vec,rskey);
-    if(rs == null || (computeHisto && !rs.hasHisto()))
-      fs.add(new RPC(rskey.home_node(),new ComputeRollupsTask(vec,computeHisto)).addCompleter(new H2OCallback() {
-        @Override public void callback(H2OCountedCompleter h2OCountedCompleter) {
+    if(rs == null || (computeHisto && !rs.hasHisto())) {
+      Log.debug("Will calculate RollupStats, key = ", rskey);
```

Comment:
Hm, not sure if we want to log every new rollup computation. This can fill up the logs if you have many columns. 
If we do, we should also log it at RollupStats.get() (second place where new rollups task can be launched from)